src/rpc/server.ts

```diff
@@ -769,27 +781,30 @@ export class RpcServer {
   // eslint-disable-next-line require-await
   public async simulateTransaction(
     tx: Transaction | FeeBumpTransaction,
-    addlResources?: RpcServer.ResourceLeeway
+    addlResources?: RpcServer.ResourceLeeway,
+    authMode?: Api.SimulationAuthMode,
   ): Promise<Api.SimulateTransactionResponse> {
-    return this._simulateTransaction(tx, addlResources)
+    return this._simulateTransaction(tx, addlResources, authMode)
       .then(parseRawSimulation);
   }
 
   // eslint-disable-next-line require-await
   public async _simulateTransaction(
     transaction: Transaction | FeeBumpTransaction,
-    addlResources?: RpcServer.ResourceLeeway
+    addlResources?: RpcServer.ResourceLeeway,
+    authMode?: Api.SimulationAuthMode,
   ): Promise<Api.RawSimulateTransactionResponse> {
     return jsonrpc.postObject(
       this.serverURL.toString(),
       'simulateTransaction',
       {
         transaction: transaction.toXDR(),
+        authMode: authMode ?? "",
```

Comment:
I don't think need to coerce to empty string as null should translate to same zero value(empty string) when rpc unmarshals.